.github/workflows/script-evaluation-test.yml

```diff
@@ -0,0 +1,37 @@
+# A nightly job which downloads script evaluation dumps from S3 and runs a regression test.
+name: Script Evaluation Test
+on:
+  schedule:
+    - cron: 30 3 * * * # 3:30am every day 
+
+jobs: 
+  script-evaluation-test: 
+    runs-on: [self-hosted, plutus-benchmark]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3.3.0
+
+      - name: Download and Unzip Dump Files
+        run: |
+          export LOCAL_DIR="$HOME/mainnet-script-dump-downloaded" 
+          nix develop --no-warn-dirty --accept-flake-config --command \
+            bash ./scripts/s3-sync-unzip.sh s3://plutus/mainnet-script-dump/ \*.event.bz2
+        env: 
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          AWS_DEFAULT_REGION: us-east-1
+          AWS_ENDPOINT_URL: https://s3.devx.iog.io
```

Comment:
Can someone confirm that these two are correct? `AWS_ENDPOINT_URL` and `AWS_DEFAULT_REGION`
Maybe @Pacman99